src/application/background-script.ts

```diff
@@ -25,6 +29,15 @@ browser.runtime.onInstalled.addListener(({ reason }) => {
     switch (reason) {
       //On first install, open new tab for onboarding
       case 'install':
+        // /!\ skip onboarding in test env
+        if (process.env.NODE_ENV === 'test') {
+          marinaStore.dispatch(setWalletData(testWalletData));
+          marinaStore.dispatch(enableWebsite('louisinger.github.io', 'regtest')); // skip the enable step too
```

Comment:
I would host this on a more "private" domain @tiero ?

Reply:
wdym by "private"? on VV repos?

Reply:
you can transfer the ownership of the repository